vpr/src/base/globals.h

```diff
@@ -9,4 +9,20 @@
 
 extern VprContext g_vpr_ctx;
 
+typedef enum UserMoveOutcome {
+    MOVE_REJECTED,
+    MOVE_ACCEPTED
+} t_user_move_outcome;
+
+struct ManualMoveInfo {
+    int block_id;
+    int to_x;
+    int to_y;
+    int subtile = 0;
+    double delta_c = 0;
+    double bb_delta_c = 0;
+    double timing_delta_c = 0;
+    t_user_move_outcome move_outcome;
+};
```

Comment:
You should probably not put this in ```globals.h```. Maybe in another header file.

Reply:
Yup, globals.h is for really major data structures everybody sees and really just includes the contexts now.  This definitely should not go in globals.h.

Reply:
resolved